docs/framework/index.md

```diff
@@ -0,0 +1,22 @@
+--
+title: .NET Framework
+description: .NET Framework
+keywords: .NET, .NET Server, .NET Desktop
+author: dotnet-bot
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net
+ms.technology: .net
+ms.devlang: dotnet
+ms.assetid: a27b2ae4-154e-4b2b-b221-0c4c05185274
+---
+
+# .NET Framework
+
+Under this section you'll find information about the .NET Framework
+on Windows. 
+
+This first important part of the story is that you can now run
+your .NET applications in [Docker containers on Windows](docker/index.md).
```

Comment:
Maybe we need to clarify this with the container team? I found the usage a bit odd on @BillWagner 's article as well and maybe it's just that I'm mistaken.

To the best of my knowledge,
Windows containers is the product and docker is the tooling around managing them. Which is unlike Linux where you have different kinds of containers like LXC containers and Docker containers.

If I'm to be extremely pedantic, I think we should refer to them as Windows containers running on the docker engine. (All this is assuming my assumptions are correct 😛 )

Let's clear this up and be consistent all across our docs.